t/Makefile.am

```diff
@@ -1,4 +1,4 @@
 AM_MAKEFLAGS = PERL5OPT="-MDevel::Cover=-db,$(abs_builddir)/cover_db"
-TESTS = 00-compile-check-all.t 01-test_needle.t 02-test_ocr.t 03-testapi.t 04-check_vars_docu.t 05-pod.t 06-pod-coverage.t 07-commands.t 08-autotest.t 09-lockapi.t 10-terminal.t 11-image-ppm.t 12-bmwqemu.t 13-osutils.t 20-openqa-benchmark-stopwatch-utils.t 99-full-stack.t
+TESTS = 00-compile-check-all.t 01-test_needle.t 02-test_ocr.t 03-testapi.t 04-check_vars_docu.t 05-pod.t 06-pod-coverage.t 07-commands.t 08-autotest.t 09-lockapi.t 10-terminal.t 11-image-ppm.t 12-bmwqemu.t 13-osutils.t 14-isotovideo.t -20-openqa-benchmark-stopwatch-utils.t 99-full-stack.t
```

Comment:
if you removed the - before the 20 I would be fine to merge

Reply:
done, updated. For the reference for later: I don't mind if we group the test variables better but I suggest we come up with a convention first covering all openQA, os-autoinst and os-autoinst-distri-opensuse and don't try to fix it by discussing about a single variable (and then never end and never proceed).